api/controller.py

```diff
@@ -759,6 +759,21 @@ def _crawlable_feed(self, library, title, url, lane, annotator=None):
         )
         return feed_response(feed)
 
+    def _load_search_facets(self, lane):
+        entrypoints = list(flask.request.library.entrypoints)
+        if len(entrypoints) > 1:
+            # There is more than one enabled EntryPoint.
+            # By default, search them all.
+            default_entrypoint = EverythingEntryPoint
+        else:
+            # There is only one enabled EntryPoint,
+            # and no need for a special default.
+            default_entrypoint = None
+        return load_facets_from_request(
+            worklist=lane, base_class=SearchFacets,
+            default_entrypoint=default_entrypoint,
+        )
+
```

Comment:
The bit of test_controller.py that uses this method in `assert_propagates_facets` isn't in this branch branch, so this method is only used once here.